text.js

```diff
@@ -185,6 +185,10 @@ define(['module'], function (module) {
             //Load the text. Use XHR if possible and in a browser.
             if (!hasLocation || useXhr(url, defaultProtocol, defaultHostName, defaultPort)) {
                 text.get(url, function (content) {
+                    if (config && config.isBuild && config.onBuildRead) {
```

Comment:
This seems to be most logical place to put the onBuildRead hook, since it closes over all of the important variables starting with `config`.